sdk/Lusid.Sdk.Tests/Features/LusidFeature.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace Lusid.Sdk.Tests.Features
+{
+    public sealed class LusidFeature : Attribute
+    {
+        public LusidFeature(string code)
+        {
+            Code = code;
```

Comment:
add validation for non-empty strings

Reply:
Turns out that if I put empty string validation into the LusidFeature attribute class itself, I am then unable to unit test this, because an exception gets thrown whenever the test loads. I think I will keep it in the "ValidateFeatures" method within the "FeatureExtractor" class, which is also a centralised place that contains all feature validation checks. What do you think?

Reply:
I think its better keep the validation in `LusidFeature`. That will also mean you don't need your empty string/null tests as it won't be possible to create them in the first instance (the validation will prevent them)

Reply:
Ok, great! I'll remove the tests for that then